bots/discord/ree6/README.md

```diff
@@ -0,0 +1,62 @@
+# Ree6
+
+___
+
+## Authors / Contributors
+
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<table>
+    <tr>
+        <td align="center">
+            <a href="https://github.com/dxssucuk">
+                <img src="https://avatars.githubusercontent.com/u/53257574" width="50px;" alt=""/><br /><sub><b>Presti</b></sub>
+            </a>
+            <br />
+            <a href="https://github.com/Ree6-Applications/Ree6/commits?author=dxssucuk" title="Codes">💻</a>
+            <a href="https://github.com/Ree6-Applications/Ree6/commits?author=dxssucuk" title="Original Bot Creator">🤖</a>
+        </td>
+        <td align="center">
+            <a href="https://github.com/parkervcp">
+                <img src="https://avatars.githubusercontent.com/u/1207679" width="50px;" alt=""/><br /><sub><b>parkervcp</b></sub>
+            </a>
+            <br />
+            <a href="https://github.com/parkervcp/eggs/commits?author=parkervcp" title="Codes">💻</a>
+            <a href="https://github.com/parkervcp/eggs/commits?author=parkervcp" title="Original Egg Creator">🥚</a>
+        </td>
+    </tr>
+</table>
+<!-- markdownlint-enable -->
+<!-- prettier-ignore-end -->
+
+___
+
+## Bot Description & Features
+
+From the developer's [Github](https://github.com/Ree6-Applications/Ree6):
+> Ree6 is an all-in-one Discord Bot maintained by Presti, which started as an alternative of Mee6 to provide a lot of features that are hidden behind a paywall or subscription.
+
+- Contains almost every feature Mee6 has.
+- Has a open Source Webinterface.
+- Provides advanced Audit-Logging.
+- Has built in post notifications (Twitter, Instagram, YouTube, Reddit and Twitch).
+- Built-in simple Ticket System.
+- And more!
+___
+
+## Configuration
+
+- You will need to modify the config.yml to even start the bot.
+
+___
+
+## Server Ports
+
+There are no ports required for Ree6.
```

Comment:
A pterodactyl server will always have at least 1 port allocated even if the application doesn't use it

Reply:
Yeah but Ree6 itself does not require any ports.
A user still has to add one but its not needed by Ree6.